framework/source/class/qx/bom/element/AnimationHandle.js

```diff
@@ -131,6 +132,11 @@ qx.Bootstrap.define("qx.bom.element.AnimationHandle",
         this.stopped = true;
         qx.bom.element.AnimationJs.stop(this);
       }
+    },
+
+    __setEnded : function(){
```

Comment:
I though it was not needed since it was a private function. I will promptly fix it.